### PR TITLE
task 4: directive and pipes tasks

### PR DESCRIPTION
**Pull request by mukeshgulia**

### CODE REVIEW

File: my-app/src/app/core/pipes/order-by.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Course } from 'src/app/models/course';
+
+@Pipe({
+  name: 'orderBy'
+})
+export class OrderByPipe implements PipeTransform {
+
+  public transform(courses: Course[]): unknown {
```
Comment (kirillpanov) [inline my-app/src/app/core/pipes/order-by.pipe.ts]: transform method returns also array of courses

File: my-app/src/app/main-layout/content-projection/course/course.component.html [hunk 2]
```diff
@@ -1,12 +1,20 @@
-<div class="container">
+<div class="container"
+  [appHighlight] = "course"
+  [ngClass]="{'course__topRated': course.topRated}">
+
   <div class="row">
     <!-- Task 2  {{object.property}}-->
     <div class="col-lg">
     <!-- Task 2  Details-->
+<<<<<<< HEAD
```
Comment (kirillpanov) [inline my-app/src/app/main-layout/content-projection/course/course.component.html]: please, resolve merge conflicts

File: my-app/src/app/models/course.ts [hunk 3]
```diff
@@ -7,6 +7,7 @@ export class Course implements CourseInterface {
   public creationDate: Date;
   public duration: number;
   public description: string;
+  public topRated: boolean;
```
Comment (kirillpanov) [inline my-app/src/app/models/course.ts:10]: should also be a property in courseList interface

**Review by kirillpanov — APPROVED** [review]

File: my-app/src/app/main-layout/content-projection/course/course.component.html (hunk 2 above)
Comment (mukeshgulia) [reply my-app/src/app/main-layout/content-projection/course/course.component.html]: done

File: my-app/src/app/core/pipes/order-by.pipe.ts (hunk 1 above)
Comment (mukeshgulia) [reply my-app/src/app/core/pipes/order-by.pipe.ts]: done

File: my-app/src/app/models/course.ts (hunk 3 above)
Comment (mukeshgulia) [reply my-app/src/app/models/course.ts:10]: done

**Pull request merged**